packages/fuselage/src/components/Box/AnimatedVisibility/index.js

```diff
@@ -48,7 +53,7 @@ export function AnimatedVisibility({ children, visibility: propVisibility }) {
     return null;
   }
 
-  return <PropsProvider children={children} />;
+  return <PropsProvider ref={ref} children={children} />;
```

Comment:
The `ref` prop must be set on the function of `useProps` hook. Every prop defined that will be transfered to the children `Box`'es.